src/translations/fixVariables.js

```diff
@@ -0,0 +1,192 @@
+const fs = require('fs');
```

Comment:
what is the purpose of this script?

Reply:
This is to address a recent bug introduced when translating the contents of prompts: https://github.com/bitshares/beet/issues/252#issuecomment-1613992908

The machine learning based translations were accidentally translating the variables stored in locale strings `text {variable} text` 

This script extracts the variables per deep nested key from the english locale files, then compares and overwrites the variables in non english locale files.

So if we run into this issue again we can run this script in this folder via the CLI to fix the locales.

This would likely not be an issue if we had manual translations, or used a paid service, however this script fixes the issue we ran into. It's not called from the Beet client at any point.

Reply:
Example script output in PR #257 https://raw.githubusercontent.com/bitshares/beet/97446700ddc093a77df4fa9f34e8a75ec89eae5a/src/translations/fixedVariables.json

Reply:
Can you give it a meaningful name and documentation please?